src/components/common/Modal/styled.js

```diff
@@ -9,30 +9,47 @@ export const StyledModal = styled(ReactModalAdapter)`
     left: 0px;
     right: 0px;
     bottom: 0px;
-    background-color: ${({ theme }) => theme.colors.unfocusedWhite};
+    background-color: ${({ theme }) => theme.colors.overlayGrey};
+    overflow: scroll;
   }
   &__content {
     position: absolute;
-    border: ${({ theme }) => theme.borders.borderModal};
     background: #fff;
-    overflow: auto;
     outline: none;
-    padding: 20px;
+    border-radius: 10px;
+    height: 90%;
+    margin-bottom: 50px;
+
+    .modal-content {
```

Comment:
You shouldn't in an ideal world use class names etc when using styled components. They should be broken down into small components that can be reused etc. Might be a bit of work now to redo that bit, if you have time have a go.

Reply:
will break the CVModal component into 2 - content and button bar